lms/djangoapps/student_account/views.py

```diff
@@ -170,72 +163,43 @@ def _third_party_auth_context(request):
     """
     context = {
         "currentProvider": None,
-        "providers": []
+        "providers": [],
+        "finishAuthUrl": None,
+        "errorMessage": None,
     }
 
-    course_id = request.GET.get("course_id")
-    email_opt_in = request.GET.get('email_opt_in')
-    redirect_to = request.GET.get("next")
-
-    # Check if the user is trying to enroll in a course
-    # that they don't have access to based on country
-    # access rules.
```

Comment:
@bradenmacdonald thanks for looking into this. Adding an embargo check in the `baskets` API method (before checking for SKUs) sounds like the right approach. Since this issue currently exists in edx-platform, fixing it seems to be outside the scope of the work you're doing here. Let me know if you agree, and I'll make sure the bug gets tracked and prioritized internally.